src/data/ACTIONS/SHARED.js

```diff
@@ -2,18 +2,18 @@ export default {
 	SPRINT: {
 		id: 3,
 		name: 'Sprint',
-		icon: process.env.PUBLIC_URL + '/icon/action/sprint.png',
+		icon: process.env.PUBLIC_URL + '/icon/action/sprint.png'
```

Comment:
Dunno what happened here. Was gonna fix it up myself, but dropping a comment instead to go with some others.